src/styles/styles.js

```diff
@@ -266,6 +266,11 @@ const styles = {
         borderWidth: 0,
     },
 
+    buttonDisabled: {
+        backgroundColor: themeColors.offline,
```

Comment:
Could you create a new `themeColor` and name it `buttonDisabledBG`? And give it a value of `colors.gray2` please? Thanks!

Reply:
Done and push that code.

Reply:
Great, thank you! Mind sharing screenshots please? It would be helpful if you updated the original comment of this PR using the template that Tim provided here: https://github.com/Expensify/Expensify.cash/pull/1073#issuecomment-750483277

Reply:
In that template, there is a section specifically for screenshots as well.

Reply:
Done please confirm.